test/express/middleware-test.js

```diff
@@ -20,6 +20,19 @@ describe('input-validation middleware tests - Express', function () {
             middleware.init('test/pet-store-swagger.yaml');
         });
     });
+    describe('initAsync function tests', async function () {
+        it('should throw an error in case the file doesn\'t exists', function () {
+            const middleware = require('../../src/middleware');
+            expect(async () => {
+                await middleware.initAsync('test/pet-store-swagger1.yaml');
+            }).to.throw;
+        });
+        it('should resolve without formats', async function () {
+            const rewire = require('rewire');
+            const middleware = rewire('../../src/middleware');
+            await middleware.initAsync('test/pet-store-swagger.yaml');
```

Comment:
Can some kind of meaningful assertion be added here?

Reply:
added try catch with assertion on errors